pandas/tools/util.py

```diff
@@ -61,6 +61,13 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : str, numpy.dtype, or Python dtype, default None
```

Comment:
@jreback : Also, you should have made this comment beforehand instead of what you said <a href="https://github.com/pydata/pandas/pull/13425#discussion_r68858982">here</a>.